### PR TITLE
fix: id:50158 changes in accordion component for fulfill accessibility

### PR DESCRIPTION
**Pull request by Farhan-emgage**
changes in accordion component for fulfill accessibility

### CODE REVIEW

**Comment by emgageadmin** [comment]
https://atera.tpondemand.com/entity/50158

File: src/components/Accordion/Accordion.scss [hunk 1]
```diff
@@ -19,6 +19,10 @@
     font-weight: font-weight(bold);
     padding: 10px 12px;
     position: relative;
+    margin:0px;
+    display: flex;
+    justify-content: left;
+    color: #212121;
```
Comment (weegrom) [inline src/components/Accordion/Accordion.scss:25]: @Farhan-emgage this should use the Heading or DisplayText component and not require custom styling.

File: src/components/Accordion/AccordionItem.tsx [hunk 2]
```diff
@@ -51,10 +52,10 @@ class AccordionItem extends React.PureComponent<Props, never> {
     );
 
     return (
-      <div className={containerClass} style={ active ? { overflow: 'visible', height: 'auto' } : { overflow: 'hidden', height: 38 } }>
-        <div className={headerClass} style={style} onClick={this.clickHandler}>
+      <div className={containerClass}>
+        <Button componentClass={headerClass} componentStyle={style} onClick={this.clickHandler} plain>
           {header}
```
Comment (weegrom) [inline src/components/Accordion/AccordionItem.tsx:57]: @Farhan-emgage this should use the Heading or DisplayText component.

**Review by weegrom — CHANGES_REQUESTED** [review]

**Comment by weegrom** [comment]
@Farhan-emgage looks like this is still waiting on the requested changes.

**Comment by Farhan-emgage** [comment]
Actually for this discussion we suppose to arranged meeting (which was cancel due to my son health).

**Pull request closed without merging**